gluten-substrait/src/main/scala/org/apache/spark/sql/execution/ColumnarShuffleExchangeExec.scala

```diff
@@ -161,6 +162,10 @@ case class ColumnarShuffleExchangeExec(
   override def doExecuteColumnar(): RDD[ColumnarBatch] = {
     if (cachedShuffleRDD == null) {
       cachedShuffleRDD = new ShuffledColumnarBatchRDD(columnarShuffleDependency, readMetrics)
+      if (getResourceProfile.isDefined) {
+        log.info(s"Set resource profile for $child to ${getResourceProfile.get}")
```

Comment:
If shuffle has a log for the feature, do we need to add a log for broadcast (or transformer) for consistency?

Reply:
Will address it soon.